.github/workflows/run-tests.yml

```
@@ -98,5 +103,18 @@ jobs:
+          # sudo sysctl fs.inotify.max_user_watches=655360
+          # mkdir -p logs
+          # strace -f -tt -s 256 -e trace=execve,%file -o logs/bolt-test.strace.log bolt test || true
           bolt test
 
+      - name: Upload strace log
+        if: always()
+        uses: actions/upload-artifact@v4
+        with:
+          name: bolt-test-strace-log
+          path: logs/bolt-test.strace.log
+
```

YAMLlint (1.37.1)

[error] too many blank lines (1 > 0)

(empty-lines)